src/Swarrot/Broker/MessageProvider/MessageProviderInterface.php

```diff
@@ -13,6 +13,14 @@
      */
     public function get();
 
+    /**
+     * consume
+     *
+     * @param string $consumerTag Identify the consumer for the rabbitmq-server
```

Comment:
The Consumer class currently generates a random value here. If that's the intended usage, PeclPackageMessageProvider should do it internally, removing that argument from the interface IMO.